client/commands/v2/pysa_server.py

```diff
@@ -357,19 +123,9 @@ async def process_open_request(
             raise json_rpc.InvalidRequestError(
                 f"Document URI is not a file: {parameters.text_document.uri}"
             )
-        self.state.opened_documents.add(document_path)
-        LOG.info(f"File opened: {document_path}")
 
-        # Attempt to trigger a background Pyre server start on each file open
-        if not self.pyre_manager.is_task_running():
-            await self._try_restart_pyre_server()
-        else:
-            document_diagnostics = self.state.diagnostics.get(document_path, None)
-            if document_diagnostics is not None:
-                LOG.info(f"Update diagnostics for {document_path}")
-                await _publish_diagnostics(
-                    self.output_channel, document_path, document_diagnostics
-                )
+        await _publish_diagnostics(self.output_channel, document_path, [])
```

Comment:
Why have these two functions called in sequence? It seems like it'd be cleaner to just have one function to call

Reply:
There was an issue where the errors after publishing, would not clear up automatically once I corrected them. (The red squiggly line would not go away). I was missing a call to publish function with empty arguments (which clears the errors), which was there originally but got lost during the refactoring. I suppose it'll be better to add this call inside `update_errors()` itself? Just one function to call then.